editor/js/modules/webWorker.js

```diff
@@ -128,7 +128,7 @@ function startWorker(marker, video, input_width, input_height, canvas_draw) {
 
       importScripts(artoolkitUrl)
 
-      var onLoad = function () {
+      const onLoad = function () {
 
         ar = new ARController(msg.pw, msg.ph, param);
         var cameraMatrix = ar.getCameraMatrix();
```

Comment:
I don't know if we need cameraMatrix, i think we have that one from the ArControllerComponent, right?